prow/config.yaml

```diff
@@ -712,3 +712,20 @@ periodics:
     - name: cache-ssd
       hostPath:
         path: /mnt/disks/ssd0
+
+- name: ci-test-infra-triage
```

Comment:
Probably needs to be indented two spaces. go-yaml won't complain though, it will just silently drop it.